[2025-06-06T23:37:34.561-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: taxi_pipeline.train_model manual__2025-06-07T02:33:50.811318+00:00 [queued]>
[2025-06-06T23:37:34.564-0300] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: taxi_pipeline.train_model manual__2025-06-07T02:33:50.811318+00:00 [queued]>
[2025-06-06T23:37:34.564-0300] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-06-06T23:37:34.643-0300] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): train_model> on 2025-06-07 02:33:50.811318+00:00
[2025-06-06T23:37:34.651-0300] {standard_task_runner.py:57} INFO - Started process 66206 to run task
[2025-06-06T23:37:34.656-0300] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'taxi_pipeline', 'train_model', 'manual__2025-06-07T02:33:50.811318+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/taxi_dag.py', '--cfg-path', '/tmp/tmp8j5aju3f']
[2025-06-06T23:37:34.657-0300] {standard_task_runner.py:85} INFO - Job 48: Subtask train_model
[2025-06-06T23:37:34.763-0300] {task_command.py:410} INFO - Running <TaskInstance: taxi_pipeline.train_model manual__2025-06-07T02:33:50.811318+00:00 [running]> on host alex-MS-7D46
[2025-06-06T23:37:34.981-0300] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='taxi_pipeline' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2025-06-07T02:33:50.811318+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-07T02:33:50.811318+00:00'
[2025-06-06T23:37:34.986-0300] {logging_mixin.py:150} WARNING - 2025/06/06 23:37:34 WARNING mlflow.utils.autologging_utils: MLflow sklearn autologging is known to be compatible with 0.24.1 <= scikit-learn <= 1.6.1, but the installed version is 1.7.0. If you encounter errors during autologging, try upgrading / downgrading scikit-learn to a compatible version, or try upgrading MLflow.
[2025-06-06T23:38:14.843-0300] {local_task_job_runner.py:225} INFO - Task exited with return code Negsignal.SIGKILL
[2025-06-06T23:38:17.251-0300] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
